### DISCUSSION TITLE
Micropython-pybricks implementation documentation

### DISCUSSION THREAD

**Discussion by Gcopper1984** [pybricks/support#517] [Q&A] [answered]
Good morning
I am starting digging into pybricks and python programming and I think is very cool and powerful. But I noticed that some advanced topics are not explained in documentation accessible via the online editor.

For example all the "Micropython" section do not have any example.

Where I can get more example about micropython programming and that functions not related to lego things?

Thank you.

**Comment by dlech** [comment, marked as answer]
Many of functions in the `MicroPython` section are based on standard Python. So learning about Python in general is a good start. Looking at the Pybricks example projects should also give you an idea of what the most useful functions are.

**Reply by Gcopper1984** [reply]
Hi
   
   ok, thank you!

Regards
